models/calendar.yml

```diff
@@ -236,7 +236,20 @@
     - 8:45 PM:
         type: Talk
         emoji: 🚗
-        title: TBD
+        title: | 
+        Become a detective! Invesgiate how to use poorly documented APIs to ship faster!
+        description: |
+        You’re so pumped. You found an API that will bring 
```

Comment:
And it looks like this block of text (242-252) needs to be indented once more as well!